### PR TITLE
Update Dependencies not requiring code changes - Closes #2498

### LINKED ISSUE #2498: Update Dependencies not requiring code changes
### Expected behavior

Dependencies with newer minor/patch versions or non breaking changes should be upgraded to their latest versions.

- bluebird	3.5.1 -> 3.5.2
- body-parser	1.18.2 -> 1.18.3
- change-case	3.0.1 -> 3.0.2
- colors	1.1.2 -> 1.3.2
- commander	2.9.0 -> 2.19.0
- compression	1.7.1 -> 1.7.3
- cors		2.8.3 -> 2.8.5
- deep-diff	1.0.1 -> 1.0.2
- ejs		2.5.6 -> 2.6.1 (seems to not be in use anymore could be removed)
- express	4.16.0 -> 4.16.4
- extend	3.0.1 -> 3.0.2
- find		0.2.7 -> 0.2.9
- js-yaml	3.10.0 -> 3.12.0
- newrelic	4.8.1 -> 4.9.0
- pg-monitor	0.9.0 -> 1.1.0
- redis		2.7.1 -> 2.8.0
- rimraf	2.6.1 -> 2.6.2 (seems to not be in use anymore could be removed)
- semver	5.3.0 -> 5.6.0

### Actual behavior

Some dependencies are outdated

### Steps to reproduce

run `npm outdated -l`

### Which version(s) does this affect? (Environment, OS, etc...)
1.2.0

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

Some dependencies were out of date and the version jump didn't introduce breaking changes.

### How did I fix it?

Check change logs of dependencies and upgraded the ones with no breaking changes.

### How to test it?

- Run test suit
- Check build passes

### Review checklist

* The PR resolves #2498
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,16 +2,11 @@
 	"name": "lisk",
 	"version": "1.4.0-alpha.0",
 	"description": "Lisk blockchain application platform",
-	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"author":
+		"Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
```
Comment (nazarhussain) [inline package.json]: @pablitovicente You saved a lot of tedious thing for everyone. 👍

File: package.json [hunk 2]
```diff
@@ -40,45 +36,45 @@
...
 		"json-refs": "=3.0.12",
 		"lisk-elements": "=1.0.0",
 		"lodash": "=4.17.11",
 		"lisk-newrelic": "LiskHQ/lisk-newrelic#1ea09b4",
 		"method-override": "=2.3.10",
-		"newrelic": "=4.8.1",
-		"pg-monitor": "=0.9.0",
+		"newrelic": "=4.10.0",
+		"pg-monitor": "=1.1.0",
 		"pg-promise": "=8.2.1",
```
Comment (nazarhussain) [inline package.json]: Please update this package as well.

Comment (nazarhussain) [inline package.json]: Please update this package as well.

File: package.json [hunk 3]
```diff
@@ -40,45 +36,45 @@
...
+		"js-yaml": "=3.12.0",
 		"json-refs": "=3.0.12",
 		"lisk-elements": "=1.0.0",
 		"lodash": "=4.17.11",
 		"lisk-newrelic": "LiskHQ/lisk-newrelic#1ea09b4",
 		"method-override": "=2.3.10",
-		"newrelic": "=4.8.1",
-		"pg-monitor": "=0.9.0",
+		"newrelic": "=4.11.0",
+		"pg-monitor": "=1.1.0",
```
Comment (lsilvs) [inline package.json]: This is a major update. Is it ok to update anyway?

Comment (pablitovicente) [reply package.json]: This one was ok as the changelog does not mention breaking changes: https://github.com/vitaly-t/pg-monitor/releases

File: package.json (hunk 2 above)
Comment (pablitovicente) [reply package.json]: @nazarhussain could you please check if this comment can be resolved?

File: package.json [hunk 4]
```diff
@@ -40,45 +36,45 @@
...
-		"express": "=4.16.0",
+		"deep-diff": "=1.0.2",
+		"express": "=4.16.4",
 		"express-domain-middleware": "=0.1.0",
 		"express-query-int": "=1.0.1",
 		"express-rate-limit": "=2.8.0",
-		"extend": "=3.0.1",
-		"find": "=0.2.7",
+		"extend": "=3.0.2",
+		"find": "=0.2.9",
```
Comment (MaciejBaj) [inline package.json]: We don't use ` find` anywhere, do we?

File: package.json [hunk 5]
```diff
@@ -40,45 +36,45 @@
...
-		"ejs": "=2.5.6",
-		"express": "=4.16.0",
+		"deep-diff": "=1.0.2",
+		"express": "=4.16.4",
 		"express-domain-middleware": "=0.1.0",
 		"express-query-int": "=1.0.1",
 		"express-rate-limit": "=2.8.0",
-		"extend": "=3.0.1",
-		"find": "=0.2.7",
+		"extend": "=3.0.2",
```
Comment (MaciejBaj) [inline package.json]: We can eliminate `extend` dependency with replacing it with Object.assign

File: package.json (hunk 4 above)
Comment (pablitovicente) [reply package.json]: It's being used on parallel tests will move to dev dependencies.

File: package.json (hunk 5 above)
Comment (pablitovicente) [reply package.json]: Eliminated and replaced with Object.assign calls as we didn't have uses where Object.assign behaves differently.

File: package.json (hunk 2 above)
Comment (pablitovicente) [reply package.json]: @nazarhussain @MaciejBaj updated pg-promise to latest minor version (changelog did mention a breaking change but for a feature we're not using)

**Pull request merged**